rfcs/rfc-incorporation-application-staff-filing.md

```diff
@@ -0,0 +1,31 @@
+- Start Date: 2022-05-05
+- Target Major Version: 
+- Entity Issue: [9248](https://github.com/bcgov/entity/issues/9248)
+- Implementation PR: (leave this empty)
+
+# Summary
+
+This RFC outlines how the changes required for the staff to file an incorporation application. The UI flow
+from the staff dashboard (how to launch the Incorporation Application/Registration) is not in the scope of
+this RFC.
+
+# Detailed design
+
+For a registration/incorporation, the temporary registration is created using an account and the temp registration
+is affiliated to the account. Once the temporary registration is converted to actual business, the business 
+is affiliated to the account. If the temp registration and business are not affiliated to an account, the business
+will not be accessible from the dashboard.
+
+The suggestion is to use a specific account( as in PPR) for staff incorporation applications. The account id can be 
+stored as a config value. If the logged in user is a staff, 'Create Named or Numbered Company' will use this 
+account id for creating temporary registration, affiliation etc. 
```

Comment:
Please clarify that end-to-end flow should work:
1. create NR if needed
2. use NR to create a named incorporation (IA) or firm (registration)
3. able to click Create Numbered Company

Reply:
Create NR is something staff can do anyways. There has to be some UI changes done to use this account for a staff user.